C++ Solution to USACO Bronze 2017 December - Milk Measurement (#2472)
cpinitiative/usaco-guide: solutions/usaco-761.mdx

	vector<tuple<int, string, int>> list;
	for (int i = 0; i < num_of_inputs; i++)
	{
		int temp1;
		string temp2;
		int temp3;
		cin >> temp1 >> temp2 >> temp3;
		list.push_back(make_tuple(temp1, temp2, temp3));
	}
	/*
	 * Sort list by first column.
	 * More information on sorting can be found on https://usaco.guide/bronze/intro-sorting
	 */
	sort(list.begin(), list.end());
	/*
	 * Map which holds the current production of each cow.
	 * In the future, we will change the production values based on input.
	 */
	map<string, int> cows = {
		{"Bessie", 7},
		{"Elsie", 7},
		{"Mildred", 7}};
	/*
	 * Store current winner inside a pair,
	 * noting the highest number and a string with cow names on the award.
	 */
	pair<int, string> winner;
	winner.first = 7;
	/*
	 * Using a string, we can efficiently store current winners
	 * and easily compare them to past winners.
	 */
	winner.second = "BessieElsieMildred";
	// Variable that will detect how many times the display changes
	int display_changes = 0;
	/*
	 * Iterate through number of entries in list.
	 * Remember, list was sorted earlier,
	 * meaning we don't need to worry about order of events.
	 */
	for (int i = 0; i < num_of_inputs; i++)
	{

> 		// Earlier I said we would be changing production values based on input. That happens here!

Reviewer: ```suggestion
		// Change production values based on input as mentioned previously.
```
